lib/bundles.js

```diff
@@ -35,6 +35,10 @@
 
       if (!(load.name in loader.defined))
         for (var b in loader.bundles) {
+          if (load.name.indexOf(b) >= 0) {
+            // Don't explore yourself
+            break;
+          }
```

Comment:
This isn't a valid check unfortunately, as `b` is in the unnormalized module name space, while `load.name` is in normalized module name space. Comparisons can only be done between fully normalized modules.